code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -592,9 +592,13 @@
 //water electrolysis
 /datum/chemical_reaction/electrolysis
 	results = list(/datum/reagent/oxygen = 1.5, /datum/reagent/hydrogen = 3)
```

Comment:
Electrolysing 5 moles (units = moles and are treated as such) of water would result it 5 moles of hydrogen (we assume hydrogen is H2) and 2.5 moles of oxygen (we assume oxygen is O2).

Reply:
okie, I just copied the existing electrolysis reaction